app/.env.example

```
@@ -1 +1,2 @@
-TRANSLATE_API_KEY=
\ No newline at end of file
+TRANSLATE_API_KEY=
+TRANSLATE_LOCATION=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)